includes/Hooks.php

```diff
@@ -2,19 +2,22 @@
 
 namespace Miraheze\CreateWiki;
 
-use DatabaseUpdater;
 use EchoAttributeManager;
 use MediaWiki\MediaWikiServices;
 use Miraheze\CreateWiki\Notifications\EchoCreateWikiPresentationModel;
 use Miraheze\CreateWiki\Notifications\EchoRequestCommentPresentationModel;
 use Miraheze\CreateWiki\Notifications\EchoRequestDeclinedPresentationModel;
 
-class Hooks {
+class Hooks implements
+	\MediaWiki\Hook\SetupAfterCacheHook,
+	\MediaWiki\Installer\Hook\LoadExtensionSchemaUpdatesHook
+{
 	public static function getConfig( string $var ) {
 		return MediaWikiServices::getInstance()->getConfigFactory()->makeConfig( 'createwiki' )->get( $var );
```

Comment:
This can be injected using `services` in HookHandlers now I believe. If you'd rather not do that we don't have to, just an idea.

Reply:
I've pushed https://github.com/miraheze/CreateWiki/pull/310/commits/b5420ee84d20e15134637dc3953d882c45b02589 for it, thank you.